web/client/codechecker_client/cmd/store.py

```diff
@@ -579,6 +581,49 @@ def parse_report_files(report_files: Set[str], zip_iter=map):
             missing_source_files)
 
 
+def get_blame_info(repo: Repo, file_path: str):
+    """ Get blame info for the given file in the given git repo. """
+    tracking_branch = None
+    try:
+        # If a commit is checked out, accessing the active_branch member will
+        # throw a type error. In this case we will use the current commit hash.
+        tracking_branch = str(repo.active_branch.tracking_branch())
+    except Exception:
```

Comment:
I changed it to `TypeError` and I hope no other excetpions will be raised here :)

Reply:
Are we expecting an exception for the case when `repo` object doesn't have an `active_branch` member? Because in this case an `AttributeError` is thrown.

Reply:
I think every `repo` has an `active_branch` member, so this is not a use case.